herbert_core/utilities/strings/stringmatchi.m

```diff
@@ -1,39 +1,58 @@
-function ind = stringmatchi (str, strcell)
-% Index of match(es) or unambiguous abbreviations in a cell array of strings
+function ind = stringmatchi (str, strcell, exact)
+% Indices of matches or unambiguous abbreviations in a cell array of strings
```

Comment:
ok now I can because I've seen the tests. But two virtually identical functions differing in the type of output look as if some duplicate code could be omitted

Reply:
My refactoring has actually made stringmatchi just use stringmatchi_log; its two lines of code now.